content/blog/redfish-interoperability-profiles.md

```diff
@@ -0,0 +1,239 @@
+
+![Manager required properties](/img/fig2-managerrequiredproperties.png "Manager required properties")
+
+<figcaption>Figure 2: Manager required properties</figcaption>
+
+Requiring the implementation of only a small number properties *provide\[s] significant flexibility, and allow conforming implementations on a wide variety of products* as mentioned in the abstract of the <a href="https://www.dmtf.org/sites/default/files/standards/documents/DSP0272_1.8.0.pdf" target="_blank">standard document</a>
+
+### Who cares?
+
+Flexibility is great, but too much flexibility can be a problem for standard organizations, sofware projects or end users willing to move from the Intelligent Platform Management Interface (<a href="https://www.intel.com/content/www/us/en/products/docs/servers/ipmi/ipmi-home.html" target="_blank">IPMI</a>) to Redfish for hardware management.
+
+Concerning standard organizations and software projects, the <a href="https://www.opencompute.org/about" target="_blank">Open Compute Project®</a> (OCP) and the <a href="https://wiki.openstack.org/wiki/Main_Page" target="_blank">OpenStack</a> projects can only adopt Redfish as their preferred management technique if they can easily define some sort of "baseline" containing which property must, should or could be implemented in their managed nodes.
+
+Concerning OCP, the charter of the <a href="https://www.opencompute.org/projects/hardware-management" target="_blank">Hardware Management Project</a> mentions: "*The hardware management specification incorporates \[...] tools and best practices \[...] for remote machine management*". This means that any server compliant to this specification must implement the network protocol(s) mentioned in the baseline.
+
+Systems eligible by the OpenStack <a href="https://wiki.openstack.org/wiki/Ironic" target="_blank">Ironic</a> (bare metal machine provisioning) and the <a href="https://wiki.openstack.org/wiki/Valence" target="_blank">Valence</a> projects (system lifecycle management) must implement as well a baseline of features containing at least the possibility to be started or stopped from remote.
+
+Redfish clients designed for managing multi-vendor systems have also a need for a list of mandatory and recommended features. For example, if a system cannot return the firmware version of its BMC, the client will have difficulties to perform firmware updates.
+
+## Redfish Interoperability Profiles
+
+To address the baseline issue mentioned above, the DMTF created the
+<a href="https://www.dmtf.org/dsp/DSP0272" target="_blank">DSP0272</a> specification document that defines Interoperability Profiles. A Redfish Interoperability Profile (or Profile) is a JSON document enumarating resources and properties that must, should or could be implemented in a Redfish service.
+
+### Didactic minimal Profile example
+
+The following example is a minimal profile created for didactic purpose. It is not relevant for use in a real life context.
```

Comment:
It is not relevant for use in a real life context. to replaced by  It is not relevant for use in a proper production context.